actions/graybox/initiate-promote-worker.js

```diff
@@ -187,7 +187,7 @@ async function findAllFiles(experienceName, appConfig, sharepoint) {
     return findAllGrayboxFiles({
         baseURI: sp.api.file.get.gbBaseURI,
         options,
-        gbFolders: appConfig.isDraftOnly() ? [`/${experienceName}/drafts`] : [''],
+        gbFolders: appConfig.isDraftOnly() ? [`/${experienceName}/drafts`] : [`/${experienceName}`],
```

Comment:
From what I remember, it traverses at the top level because it needs to pick the files from the `/locale/exp` folders as well. 

It further filters the path when traversing using a regex
`const pathsToSelectRegExp = new RegExp(`^/([^/]+/)?${experienceName}(/.*)?$`);`

Reply:
Makes sense. Thanks @sukamat !!
Will close the PR.